portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java

```diff
@@ -56,6 +56,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.getWriter().println(jsonComments);
   }
 
+
```

Comment:
intentional?  formatter made this?

Reply:
This was likely me by accident after running the formatter. Fixed.